### ISSUE TITLE
a11y: dashboard page needs a main element

### ISSUE THREAD

**Issue by Danielle254** [LetsGetTechnical/elecretanta#528]
Issue: '/dashboard' page needs a `<main>` element. This should come after the `<nav>` element. Relevant page content should be nested inside the `<main>` element.
